src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -19,7 +22,13 @@ const pageTypeUrls = (assetType, service, variant) => {
           name: 'secondaryColumn',
           path: getSecondaryColumnUrl({ service, variant }),
         },
-      ];
+        hasRecommendations(service, variant)
+          ? {
+              name: 'recommendations',
+              path: getRecommendationsUrl({ assetUri, variant }),
+            }
+          : null,
+      ].filter(i => i);
```

Comment:
This filters out the `null` value so that the map() which uses this doesn't iterate over that value. Not sure what that does when it tries to fetch the null value, so this might not be necessary.